Frontend/src/App.tsx

```diff
@@ -25,34 +37,37 @@ const AppBarConditional = () => {
         return null; // Zeigt nichts an, wenn die Route eine der ausgeschlossenen Routen ist
     }
 
-    return <NavBar />;
+    return null;      //return <Footer />
```

Comment:
this function always returns null

Reply:
Yes, thats correct. This function actually insn't about the return value - it just routes after checking the condition loginStatus=true/false. It works like intended. You can check yourself by switching loginStatus and try to access loggedIn-pages-only. Maybe there's a better way to write this function. If you have a recommendation pls let me know, otherwise lets just wait for the backend implementation or the performance check in the end.